### DISCUSSION TITLE
因为闪退问题，充错了会员账户，可以把充值换到之前的账户么

### DISCUSSION THREAD

**Discussion by daxiong0513** [getlantern/lantern-forum-cn#352] [Q&A] [answered]
最近因为 win11 下的 lantern 7.8.1 闪退问题，让我以为我的会员到期，结果在 3-30 充值了 2 年会员后，发现自己充错了邮箱，虽然两个邮箱都是自己的，但是因为前一个会员还有 4 个月的时长，想把这两个账户的时间合并下。以之前的会员账户为准，可以么？
@Mike777Khoo 

才充值的会员账户
![image](https://github.com/getlantern/lantern-forum-cn/assets/11566538/ffc88d55-a285-4f01-9cf6-f6dd1cf42b59)

之前的会员账户
![1147543013](https://github.com/getlantern/lantern-forum-cn/assets/11566538/a6cbe85a-c543-41c3-99b9-b33e60d22a1f)

3-30 的充值记录
![IMG_7790](https://github.com/getlantern/lantern-forum-cn/assets/11566538/265be6a9-ac7b-43ec-9207-bc20624b0184)

**Comment by MaxMKMax** [comment]
@daxiong0513 您好. 请问您要合并到一起的是哪一个邮箱呢? 
 这里有两个邮箱.
 
 我不清楚能不能合并. 我帮您问看.

**Reply by daxiong0513** [reply]
合并到 daxiong0513@qq.com就行了。

**Reply by MaxMKMax** [reply]
@daxiong0513 
嗯嗯. 好的. 我帮您问问看是否可以帮您转移. 您给我们一点时间. 我再告知您. 您也留意邮箱和GitHub.

**Reply by daxiong0513** [reply]
有什么更新么？

**Reply by daxiong0513** [reply]
@Mike777Khoo , 我这里有什么更新么？

**Reply by MaxMKMax** [reply, marked as answer]
@daxiong0513 您好. 已经把您的账号合并在一起了. 然后重新加两年给您. 谢谢您支持蓝灯!

**Reply by daxiong0513** [reply]
收到了，谢谢~

**Reply by MaxMKMax** [reply]
@daxiong0513 
不客气呢. 谢谢您支持蓝灯!

**Comment by blueraincoatli** [comment]
@Mike777Khoo 我碰到类似问题，我原来注册和充值的邮箱 ray.lee@artspire.cn ，这个邮箱我不能使用了，而这次闪退的问题导致我无法再次登录我的专业版账号，因为拿不到验证码了，要怎么办？是否可以转移到另一个邮箱账号？blueraincoat@sina.com，曾经也是专业版账号

**Reply by MaxMKMax** [reply]
@blueraincoatli 
嗯嗯. 好的. 我帮您问问看是否可以帮您转移. 您给我们一点时间. 我再告知您. 您也留意邮箱和GitHub.

**Reply by blueraincoatli** [reply]
谢谢！

**Reply by MaxMKMax** [reply]
@blueraincoatli 您好. 我来帮您再问问看! 刚才已经在问了. 我等待回复. 您等等一下啊. 不好意思.
